client/src/components/VideoHero.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "../App.css";
+// import image1 from "../images/PladeholderImage.jpeg";
+
+function VideoHero({ src, height, width, subtitle, textOverlay }) {
```

Comment:
Hi Monika, can I check should this be part of the Header and Footer PR?

Reply:
you are right. These changes should have been in the videohero branch. I made mistake. can we still merge it, pls?

Reply:
i am goin to remove it from this branch and push it from the videostory branch